src/Revise.jl

```diff
@@ -337,6 +385,32 @@ function track(mod::Module)
         end
         # Add the files to the watch list
         process_parsed_files(files)
+    elseif mod == Core.Compiler
+        # find the Julia source dir by looking at the location of sysimg.jl
```

Comment:
What about using `Sys.BINDIR`? Or how about `Revise.basesrccache` to be a bit independent of the Julia version?

Reply:
Those don't work when working with out-of-tree builds, where both `Sys.BINDIR` and `Revise.basesrccache` will point to inside the build directory. Or how would you suggest finding the `base` dir from those paths?